lib/SPIRV/SPIRVReader.cpp

```diff
@@ -529,6 +529,9 @@ Type *SPIRVToLLVM::transType(SPIRVType *T, bool IsClassMember) {
     SmallVector<Type *, 4> MT;
     for (size_t I = 0, E = ST->getMemberCount(); I != E; ++I)
       MT.push_back(transType(ST->getMemberType(I), true));
+    for (auto &CI : ST->getContinuedInstructions())
+      for (size_t I = 0, E = CI->getNumElements(); I != E; ++I)
+        MT.push_back(transType(CI->getMemberType(I), true));
```

Comment:
May range based loop be used here ?
```suggestion
      for (auto &Elem : CI->getElements())
        MT.push_back(transType(Elem, true));
```

Reply:
It cannot be used unless `gelElements` method is added for `SPIRVTypeStructContinuedINTEL` class. I didn't add such thing to make its interface closer to interface of `SPIRVTypeStruct` class.

Reply:
Ah, sorry, missed enable_if condition.